source/_integrations/huawei_lte.markdown

```diff
@@ -91,6 +86,8 @@ To enable the component, add the following lines to your
 # Example configuration.yaml entry
 huawei_lte:
   - url: http://192.168.100.1/
+    username: ROUTER_USERNAME
+    password: ROUTER_PASSWORD
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely do not need. If required, insert a full configuration sample later that covers special setups or alike.

```suggestion
```

Reply:
Username and password are now actually required when configuring the integration for the first time, see parent PR.

Reply:
In that case, the schema below needs updating, as it shows they are optional.

Reply:
Ah, very much so, will do.